template/nuxt/nuxt.config.js

```diff
@@ -108,7 +108,12 @@ module.exports = {
       }
     },<% } %><% if (ui === 'element-ui') { %>
     transpile: [/^element-ui/],
-    <% } %>
+    <% } %> <% if (ui === 'tailwind') { %>
+    postcss: {
+      plugins: [
+        require('tailwindcss')('./tailwind.config.js')
```

Comment:
```suggestion
        tailwindcss: './tailwind.config.js'
```

See https://github.com/Developmint/developmint.de/blob/master/nuxt.config.js#L226
You can probably even emit the tailwind config part as it'll detect it automatically I'd guess 🤔 

```suggestion
        tailwindcss: {}
```